src/values/AbstractObjectValue.js

```diff
@@ -204,6 +204,49 @@ export default class AbstractObjectValue extends AbstractValue {
     return this;
   }
 
+  // ECMA262 9.1.1
+  $GetPrototypeOf(): ObjectValue | AbstractObjectValue | NullValue {
+    if (this.values.isTop()) {
+      AbstractValue.reportIntrospectionError(this);
+      throw new FatalError();
+    }
+    invariant(this.kind !== "widened", "widening currently always leads to top values");
```

Comment:
Originally I kept a path for widened values just like this one:

https://github.com/facebook/prepack/blob/master/src/values/AbstractObjectValue.js#L251-L262

But these are actually unused right now because widened values always end up as topVal. I started working on making widened values a bit more specific so that we can use this information but I'm not currently working on that so I just switched to an invariant instead for now.